src/ReportWrapper.php

```diff
@@ -21,7 +21,7 @@
 
     public function __construct($baseReport)
     {
-        $this->baseReport = is_string($baseReport) ? new $baseReport : $baseReport;
+        $this->baseReport = is_string($baseReport) ? new $baseReport() : $baseReport;
```

Comment:
```suggestion
        $this->baseReport = is_string($baseReport) ? Injector::inst()->create($baseReport) : $baseReport;
```

Reply:
That might need some spacing updates...

Reply:
Needs the namespace too- I cant accept from my phone